2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/Cloud-Classification/code/wandb/settings
2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/Cloud-Classification/code/wandb/run-20220620_130016-13vmod3u/logs/debug.log
2022-06-20 13:00:16,603 INFO    MainThread:13853 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/Cloud-Classification/code/wandb/run-20220620_130016-13vmod3u/logs/debug-internal.log
2022-06-20 13:00:16,604 INFO    MainThread:13853 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc85758ec40>
2022-06-20 13:00:16,605 INFO    MainThread:13853 [wandb_init.py:init():439] calling init triggers
2022-06-20 13:00:16,605 INFO    MainThread:13853 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'learning_rate': 3e-05, 'architecture': 'Isotropic ViG - S', 'loss': 'CrossEntropy', 'optim': 'SGD', 'dataset': 'GCD', 'epochs': 200}
2022-06-20 13:00:16,605 INFO    MainThread:13853 [wandb_init.py:init():492] starting backend
2022-06-20 13:00:16,605 INFO    MainThread:13853 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-20 13:00:16,642 INFO    MainThread:13853 [backend.py:ensure_launched():219] starting backend process...
2022-06-20 13:00:16,671 INFO    MainThread:13853 [backend.py:ensure_launched():224] started backend process with pid: 13893
2022-06-20 13:00:16,672 INFO    MainThread:13853 [wandb_init.py:init():501] backend started and connected
2022-06-20 13:00:16,690 INFO    MainThread:13853 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-06-20 13:00:16,690 INFO    MainThread:13853 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-06-20 13:00:16,691 INFO    MainThread:13853 [wandb_init.py:init():565] updated telemetry
2022-06-20 13:00:16,759 INFO    MainThread:13853 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-06-20 13:00:17,525 INFO    MainThread:13853 [wandb_run.py:_on_init():1759] communicating current version
2022-06-20 13:00:17,698 INFO    MainThread:13853 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.18 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-20 13:00:17,698 INFO    MainThread:13853 [wandb_init.py:init():625] starting run threads in backend
2022-06-20 13:00:19,198 INFO    MainThread:13853 [wandb_run.py:_console_start():1733] atexit reg
2022-06-20 13:00:19,199 INFO    MainThread:13853 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-06-20 13:00:19,200 INFO    MainThread:13853 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-06-20 13:00:19,201 INFO    MainThread:13853 [wandb_run.py:_redirect():1667] Redirects installed.
2022-06-20 13:00:19,202 INFO    MainThread:13853 [wandb_init.py:init():664] run started, returning control to user process
2022-06-20 13:00:19,215 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-20 13:00:19,215 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-20 13:00:19,244 INFO    MainThread:13853 [wandb_init.py:_resume_backend():327] resuming backend
2022-06-20 13:00:21,026 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-20 13:00:21,026 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-20 13:00:21,068 INFO    MainThread:13853 [wandb_init.py:_resume_backend():327] resuming backend
2022-06-20 13:00:21,317 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-20 13:00:21,317 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-20 13:00:21,338 INFO    MainThread:13853 [wandb_init.py:_resume_backend():327] resuming backend
2022-06-21 17:28:41,449 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-21 17:28:41,449 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-21 17:28:44,418 INFO    MainThread:13853 [wandb_init.py:_resume_backend():327] resuming backend
2022-06-21 17:28:45,608 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-21 17:28:45,608 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-21 17:40:40,099 INFO    MainThread:13853 [wandb_init.py:_resume_backend():327] resuming backend
2022-06-21 17:40:40,122 INFO    MainThread:13853 [wandb_init.py:_pause_backend():318] pausing backend
2022-06-21 17:40:40,123 INFO    MainThread:13853 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-06-21 17:40:45,237 INFO    MainThread:13853 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-06-21 17:40:45,238 INFO    MainThread:13853 [wandb_run.py:_restore():1674] restore
